legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/planner/Planner.java

```diff
@@ -104,6 +104,12 @@ default boolean collectStatistics()
             return false;
         }
 
+        @Default
+        default boolean writeStatistics()
+        {
+            return false;
```

Comment:
Curious why the default is false?

Reply:
This is for back ward compatibility with CNAS